src/test_lib_json/main.cpp

```diff
@@ -3122,9 +3122,9 @@ JSONTEST_FIXTURE_LOCAL(CharReaderStrictModeTest, dupKeys) {
   Json::CharReaderBuilder b;
   Json::Value root;
   char const doc[] =
-      "{ \"property\" : \"value\", \"key\" : \"val1\", \"key\" : \"val2\" }";
+      R"({ "property" : "value", "key" : "val1", "key" : "val2" })";
   {
-    b.strictMode(&b.settings_);
+    Json::CharReaderBuilder::strictMode(&b.settings_);
```

Comment:
These were using `b` deliberately so we can change the type of `b` without updating other lines.